frontends/ast/simplify.cc

```diff
@@ -1453,6 +1589,46 @@ bool AstNode::simplify(bool const_fold, bool at_zero, bool in_lvalue, int stage,
 			}
 			did_something = true;
 		}
+
+		// Convert only port wires
+		if ((port_id > 0)) {
+			flatten_ranges(this);
+		}
+
+		if (children.size() == 1 && children[0]->type == AST_MULTIRANGE && children[0]->is_packed) {
+			const auto* multirange = children[0];
+			const size_t ranges = std::count_if(
+				multirange->children.begin(), multirange->children.end(),
+				[](const AstNode* node) {
+					return node->type == AST_RANGE;
+				});
+			if ((ranges == multirange->children.size())) {
+				size_t size = 1;
+
+				auto* attr_ranges = new AstNode;
+				attr_ranges->type = AST_CONSTANT;
```

Comment:
```suggestion
				auto* attr_ranges = new AstNode(AST_CONSTANT);
```
I think you can set type of ``AstNode`` in the constructor directly.